### ISSUE TITLE
[0.5] 7b: Add menu to new page

### ISSUE THREAD

**Issue by amberreeves** [InteractiveMechanics/CollegeWomen_Drupal7#45]
New page (currently Links & Resources): similar to above, we will have new content that needs a new page menu.

Waiting on more info from the client on this one.

**Comment by cworthen** [comment]
@amberreeves Since the client still needs to provide feedback regarding this task, should I put on hold ?

**Comment by mtedeschi** [comment]
Yeah, you can hold on this.

**Comment by amberreeves** [comment]
i sent an email to the client asking about this, will keep you updated.
